modules/objdetect/include/opencv2/objdetect/charuco_detector.hpp

```diff
@@ -0,0 +1,154 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#ifndef OPENCV_OBJDETECT_CHARUCO_DETECTOR_HPP
+#define OPENCV_OBJDETECT_CHARUCO_DETECTOR_HPP
+
+#include "opencv2/objdetect/aruco_detector.hpp"
+
+namespace cv {
+namespace aruco {
+
+//! @addtogroup aruco
+//! @{
```

Comment:
Documentation is still broken since #22368 .

There is no `aruco` group: http://pullrequest.opencv.org/buildbot/export/pr/22986/docs/d5/d54/group__objdetect.html

Reply:
Now this looks like this:
![image](https://user-images.githubusercontent.com/22337800/209177999-d9736984-3dee-436d-8ff6-ef1c8d3f89a2.png)
Is it ok?
https://pullrequest.opencv.org/buildbot/export/pr/22986/docs/d5/d54/group__objdetect.html